### PR TITLE
support set annotation to mcm service account

### PR DESCRIPTION
**Pull request by zjj2wry**
**How to categorize this PR?**

/area TODO
/kind TODO
/priority normal
/platform aws

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator

```

### CODE REVIEW

**Review by tennix — APPROVED** [review]
LGTM

File: charts/internal/machine-controller-manager/seed/templates/serviceaccount.yaml [hunk 1]
```diff
@@ -3,3 +3,7 @@ kind: ServiceAccount
 metadata:
   name: machine-controller-manager
   namespace: {{ .Release.Namespace }}
+{{- if .Values.annotations }}
+  annotations:
+{{ toYaml .Values.annotations | indent 4 }}
```
Comment (aylei) [inline charts/internal/machine-controller-manager/seed/templates/serviceaccount.yaml:8]: Use `serviceaccountAnnotations` may be more specific

Comment (zjj2wry) [reply charts/internal/machine-controller-manager/seed/templates/serviceaccount.yaml:8]: update to serviceAccountAnnotations

**Review by tennix — APPROVED** [review]
LGTM

**Review by aylei — APPROVED** [review]
lgtm

**Pull request merged**